lucille-core/src/test/resources/KafkaTest/twoConnectors.conf

```diff
@@ -46,9 +46,6 @@ indexer {
 kafka {
   bootstrapServers: "localhost:9090"
 
-  # how long a kafka poll should wait for data before returning an empty record set
-  pollIntervalMs: 250
```

Comment:
there are several other configs in the codebase that have pollIntervalMs. Let's remove all instances as part of this PR.